### PR TITLE
embedded ngaut/systimemon

### PR DESCRIPTION
**Pull request by andelf**
Since it introduces ngaut/log. and is a one-function npm-style package.

### CODE REVIEW

**Comment by siddontang** [comment]
please move to the pkg.

**Comment by andelf** [comment]
should it be a util or whatever?

File: glide.yaml [hunk 1]
```diff
@@ -115,12 +115,10 @@ import:
   - pbutil
 - package: github.com/ngaut/deadline
   version: fae8f9dfd7048de16575b9d4c255278e38c28a4f
-- package: github.com/ngaut/log
```
Comment (siddontang) [inline glide.yaml]: seem we can't remove the log pkg here

**Comment by siddontang** [comment]
@disksing 

Is it better to send a PR to the monitor project to remove the log dependence? TiDB will use this monitor and will do the same thing later. Otherwise, TiDB will use the monitor pkg in the PD util package.

**Comment by andelf** [comment]
reject npm-style package.
"tidb will use" does not mean "tidb is using".

**Comment by siddontang** [comment]
no, TiDB is using.

**Comment by siddontang** [comment]
If a lib will be used across multi projects, it should be independent, no matter how little it is.

**Comment by andelf** [comment]
to create a new repo: pingcap/pkg with these small packages

**Comment by siddontang** [comment]
Maybe we should discuss with @ngaut at first. If he agrees on removing the log and letting the users handle it themselves, it is better.

**Comment by siddontang** [comment]
Or we can fork this repository and change it ourselves. 
IMO, I prefer using individual repositories.  /cc @coocood

**Comment by coocood** [comment]
I think fork `naugt/log` and `naugt/systimemon` under pingcap organization is better.

**Pull request closed without merging**